### ISSUE TITLE
Texture Bug: blocks become invisible when open chest!

### ISSUE THREAD

**Issue by Hasenspringer** [sp614x/optifine#6565]
## Description of Issue
Hey, I have a problem with optifine. If I open boxes etc. and move my mouse over empty storage locations, blocks become invisible. (s. screenshot) This only happens with optifine. Everything is normal in vanilla. I've reinstalled optifine several times and the error was gone for a while until it occurs again. I also get the messages opengl error 1281 and 1282 to see in the chat! I also get the message: [Render thread/WARN]: [OptiFine] Model renderer not found, (s. screenshot)

![Screenshot 2022-01-18 114758](https://user-images.githubusercontent.com/96696571/149925670-40182ce0-9166-48c7-9ac1-ed4d740f03de.png)
![Screenshot 2022-01-18 121420](https://user-images.githubusercontent.com/96696571/149927176-a5159964-d6a6-4894-ae18-d19ef17d5bf5.png)

![Screenshot 2022-01-18 120235](https://user-images.githubusercontent.com/96696571/149925674-4d92ff81-d241-4fab-bda5-b4a618e6f00f.png)

## Steps to Reproduce
1. just playing normal minecraft with optfine.

## OptiFine Version
1.18.1-OptiFine_HD_U_H4

## Installation Method
Installed via jarfile.

## Other Installed Mods
only optifine

## Log Files/Crash Reports
![Screenshot 2022-01-18 120235](https://user-images.githubusercontent.com/96696571/149926405-e91bad47-56dc-44c3-8cf5-45c02167da5f.png)

## F3 Debug Screenshot

![Screenshot 2022-01-18 121420](https://user-images.githubusercontent.com/96696571/149927208-7218cea4-44f1-40c7-a1ff-74f2c4a8c890.png)

## Prior Testing
only happens with optifine! Also when using fabric with optifine-

##other informations:
Strangely, when I press f3 and don't change anything, the error doesn't appear. It was also once the case that the error occurs when I move my mouse over empty inventory spaces.
![Screenshot 2022-01-18 121907](https://user-images.githubusercontent.com/96696571/149927848-ef79c436-ed6f-46cf-b4e2-c50f901bb915.png)

-Hannes

**Comment by Hasenspringer** [comment]
I forgot the Degub-Screen:
![Screenshot 2022-01-18 121307](https://user-images.githubusercontent.com/96696571/149928179-d54aee88-c985-4134-a69b-1442e5ae146a.png)

**Comment by KaiAF** [comment]
Does this happen without a resource pack? Try turning off antialiasing and anisotropic filtering

**Comment by Hasenspringer** [comment]
Yes it also happen without resource packs! I turned off antialising and all works fine now.
